tests/fixtures/SRP272267_upload/SRP272267_reg.xml

```diff
@@ -13,7 +13,7 @@
 			</PROJECT_ATTRIBUTE>
 			<PROJECT_ATTRIBUTE>
 				<TAG>new_study_type</TAG>
-				<VALUE>metagenome assembly</VALUE>
+				<VALUE>matagenome assembly</VALUE>
```

Comment:
Think this one was a mistake, can be changed back to metagenome?

Reply:
🤦